.github/workflows/publish.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.9]
+        python-version: ["3.10"]
```

Comment:
Will it work if someone use 3.9 to run this tool?

Reply:
Unfortunately not.

Reply:
Updated to 3.8.